transform/mattermost-analytics/models/intermediate/product/licenses/_licenses_servers/int_cloud_servers.sql

```diff
@@ -0,0 +1,12 @@
+with mm_telemetry_prod_license as (
+select timestamp::date as installation_date
+    , installation_id as installation_id
+    , server_id as server_id
+    , license_name as license_name
+from {{ ref('stg_mm_telemetry_prod__license')}} 
+where license_name in ('CLD')
```

Comment:
Do we have the `license_id`here? If yes, maybe we should use the `cloud_license_id` value - worrying that the `CLD` string is less future-proof than the license id.

Reply:
we do have `license_id` for cloud servers but its a static value, I set it in the project variables

Reply:
Yes, my q is: should we compare `license_id` against it instead of comparing against the `CLD` string.

Reply:
Yiannis is also using `installation_id != null` to determine cloud vs. non-cloud. Maybe we can use the same here or below to align on one single method.

Reply:
There are rows with `installation_id != null` they aren't `CLD` licenses, which seems weird to me. I think we need both these filters. I also couldn't map these rows to any other source to see what they are.

Reply:
Updated the filter after new input from Nick. For cloud its `installation_id != null` and self-hosted its `license_id != null and installation_id = null`